torch/_inductor/codegen/triton.py

```diff
@@ -849,7 +850,11 @@ def expm1(x):
 
     @staticmethod
     def sqrt(x):
```

Comment:
Since the dtypes are not correct currently - we should use them in codegen yet

Reply:
I see. The only reason I modified `sqrt` codegen is to have an unittest. I can remove it if it seems risky.

Reply:
Sorry, meant to type *should not. Let's still check `codegen_upcast_to_fp32` here just to be safe.